### ISSUE TITLE
Accessibility Issue: No accessible name for social media icons in footer

### ISSUE THREAD

**Issue by CorgiDev** [topcoder-platform/Training-Challenges#266]
Steps to Reproduce:

1. Open Dev Tools for browser.
2. Inspect the social media icons found in the footer section of the page.
3. There is no appropriate labeling that is accessible to users of assistive technologies.

Actual Result: 
Inspecting the logos reveals no alt text or proper aria that would make a label for it accessible to assistive technologies. When a screen reader hits the links, it just reads off the url, which not considered best practice or considered an accessible label.

Expected Result: 
Using assistive technology should cause the logo to be identified with an simple, descriptive name, like just "Topcoder Facebook" or "Topcoder Youtube".

Device/OS/Browser Information: Desktop / Windows 11 (Build 22454) / Edge (Version 94.0.992.31)

A simple solution to this would be to add the following attributes to the SVG image that makes up the logos:

- `aria-hidden="true"
- `aria-role="presentation"`
- `alt=" "`

Then add the following attribute to the link itself: `aria-label="Topcoder"`. Add additional label after "Topcoder" to identify the link's purpose. For example, "Topcoder Facebook" for the Facebook icon/link, or "Topcoder LinkedIn" for the LinkedIn logo/link.

**Comment by nithyaasworld** [comment]
@CorgiDev Can you also please update the steps as stated in the [previous bug](https://github.com/topcoder-platform/Training-Challenges/issues/265) and attach a screenshot of the issue?

**Comment by CorgiDev** [comment]
@nithyaasworld Updated with screenshots and url. I'll update the other bug the same way. Thank you for the feedback.

**Comment by nithyaasworld** [comment]
Awesome @CorgiDev 

Please keep watching challenge listing page https://www.topcoder.com/challenges?tracks[QA]=true&types[]=CH&types[]=F2F&types[]=TSK for future bug hunt challenges which hold cash prizes and start participating in it.

Join Topcoder community Slack and turn on the notification for dev-challenges channel to get immediate notification whenever a new challenge is launched. 

Good luck!
